homeassistant/components/deconz/gateway.py

```diff
@@ -161,6 +161,7 @@ def __init__(self, hass, device):
         self._device.register_async_callback(self.async_update_callback)
         self._event = 'deconz_{}'.format(CONF_EVENT)
         self._id = slugify(self._device.name)
+        _LOGGER.info("deCONZ event created: %s", self._id)
```

Comment:
This shouldn't be info, debug at best?

Reply:
You don't want to change logging level and restart just to get a hold of the event id. Some kind of accessible event registry or support in device registry would be better though